python/lib/aquad.cc

```diff
@@ -2,151 +2,281 @@
 // SPDX-License-Identifier: MIT
 
 #include "python/lib/py_wrappers.h"
```

Comment:
Includes should be modified to follow our coding style: https://open-sn.github.io/opensn/devguide/coding_standard.html#include-directives.